libqtile/widget/check_updates.py

```diff
@@ -56,29 +56,44 @@ def __init__(self, **config):
                          "Mandriva": ("urpmq --auto-select", 0)
                          }
 
-        # Check if distro name is valid.
-        try:
-            self.cmd = self.cmd_dict[self.distro][0].split()
-            self.subtr = self.cmd_dict[self.distro][1]
-        except KeyError:
-            distros = sorted(self.cmd_dict.keys())
-            logger.error(self.distro + ' is not a valid distro name. ' +
-                         'Use one of the list: ' + str(distros) + '.')
-            self.cmd = None
+        # Backwards compatibility for custom_command. Convert string to tuple.
+        if type(self.custom_command) is not tuple:
```

Comment:
`isinstance(self.custom_command, (list, tuple))` might be better here. If the user passes `['pamac checkupdates', 42]` we would get `self.custom_command = (['pamac checkupdates', 42], 0)` in the next line.

Reply:
Should I take out the deprecation message?

Reply:
Yeah, I'd suggest to drop that and the comment `# Backwards compatibility for custom_command. Convert string to tuple.` a few lines above.